.tx/config

```diff
@@ -1,33 +1,33 @@
 [main]
```

Comment:
Can you add a comment at the top linking to the reference for this file? I have no idea what the the crazy new `o:` / `p:` syntax is.